applications/FluidDynamicsHydraulicsApplication/tests/test_hydraulic_fluid_auxiliary_utilities.py

```diff
@@ -76,5 +75,18 @@ def testCalculateWettedArea(self):
         theoretical_wetted_area= 2.0*(level_set_z)
         self.assertAlmostEqual(WettedArea, theoretical_wetted_area, 12)
 
+    def testAvoidOutletInflow(self):
```

Comment:
```suggestion
    def testApplyOutletInflowLimiter(self):
```
Rename it to ease eventual code browsing.